src/Bicep.Core/PrettyPrintV2/NewlineKind.cs

```diff
@@ -10,7 +10,5 @@ public enum NewlineKind
         CRLF,
 
         CR,
-
-        Auto,
```

Comment:
This should have been removed in the original v2 formatter PR. This option was included in the version 1 formatter settings, which was initially inspired by Prettier's configuration, where `auto` was the default setting for `endOfLine` prior to Prettier 2.0.0.. However, since Prettier 2.0.0, the default `endOfLine` value was switched to `lf` in response to significant user feedback against the auto setting, as highlighted in [this discussion](https://github.com/prettier/prettier/issues/6932#issuecomment-552850131). Although Prettier maintained the `auto` option to prevent disruptive changes for existing users, the v2 formatter for Bicep is still in a preview phase. Besides, given that auto was not added to bicepconfig.schema.json, removing it at this stage should be a low-risk adjustment.